### PR TITLE
Add description for LPAR inventory

### PR DESCRIPTION
**Pull request by CallisteH**

### CODE REVIEW

**Comment by CallisteH** [comment]
For me it's ready to merge

**Review by jaywcarman — APPROVED** [review]
LGTM. @CallisteH I'm assuming this is ready to go per your comment, but will wait until your remove `[WIP]` to be sure.

File: app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb [hunk 1]
```diff
@@ -110,8 +110,8 @@ def parse_lpar_common(lpar, type)
       :ems_ref         => lpar.uuid,
       :name            => lpar.name,
       :location        => "unknown",
-      :description     => lpar.type,
       :vendor          => "ibm_power_hmc",
+      :description     => lpar.description.nil? ? "" : lpar.description,
```
Comment (chessbyte) [inline app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb:114]: You can simplify this by using the `.to_s` method available on most classes, including NilClass

```suggestion
      :description     => lpar.description.to_s,
```

Comment (CallisteH) [reply app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb]: Thanks Oleg

**Comment by CallisteH** [comment]
After a new rebase from the master, it's ready to merge for me

**Comment by jaywcarman** [comment]
Backporting to _najdorf_ requires the Gemfile.lock for `ibm_power_hmc` to be updated from `0.8.3` -> `0.8.4`
https://github.com/ManageIQ/manageiq/blob/najdorf/Gemfile.lock.release#L705

Should I submit a PR or is this comment enough?

**Comment by chessbyte** [comment]
@jaywcarman if there was a PR that bumped the gem version, then that should probably be marked with `najdorf/yes?` as well

**Comment by jaywcarman** [comment]
@chessbyte this PR does bump the minium `ibm_power_hmc` gem version in our plugin gemspec:
https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/blame/master/manageiq-providers-ibm_power_hmc.gemspec#L22

If this gets backported, does that somehow automatically propagate up to the core `manageiq/Gemfile.lock.release` in _najdorf_, or do I need to propose the change through a PR?

**Comment by agrare** [comment]
The `Gemfile.lock.release` is not automatically updated, tor example here is a PR to bump a gem in a release branch https://github.com/ManageIQ/manageiq/pull/21720
Not sure if @Fryguy wants to do the `bundle install` himself or if he wants you to put a PR in

**Comment by Fryguy** [comment]
@CallisteH A conflict occurred during the backport of this pull request to `najdorf`.

If this pull request is based on another pull request that has not been marked for backport, add the appropriate labels to the other pull request. Otherwise, please create a new pull request direct to the `najdorf` branch in order to resolve this.

Conflict details:

```diff
diff --cc manageiq-providers-ibm_power_hmc.gemspec
index dd7b427,77bed7f..0000000
--- a/manageiq-providers-ibm_power_hmc.gemspec
+++ b/manageiq-providers-ibm_power_hmc.gemspec
@@@ -19,7 -19,7 +19,11 @@@ Gem::Specification.new do |spec
    spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
    spec.require_paths = ["lib"]
  
++<<<<<<< HEAD
 +  spec.add_dependency "ibm_power_hmc", "~> 0.8.1"
++=======
+   spec.add_dependency "ibm_power_hmc", "~> 0.8.4"
++>>>>>>> 76bb863 (Merge pull request #39 from CallisteH/fix/lpar_description)
  
    spec.add_development_dependency "manageiq-style"
    spec.add_development_dependency "simplecov", ">= 0.21.2"
```

**Comment by Fryguy** [comment]
I'll manually backport this

**Comment by Fryguy** [comment]
`najdorf` backport details:

```
commit b7d0286a8fe44f2b228087715b9e716248b85e13 (HEAD -> najdorf, upstream/najdorf)
Author: Adam Grare <adam@grare.com>
Date:   Wed Feb 16 11:40:16 2022 -0500

    Merge pull request #39 from CallisteH/fix/lpar_description

    Add description for LPAR inventory

    (cherry picked from commit 76bb863c6aafaf0ca32dd690ceb22259598eebbf)
```

**Pull request merged**